products/gstreamer.md

```diff
@@ -72,23 +72,23 @@ releases:
     releaseDate: 2014-07-19
     eol: true
     # no known changelog for this versions
-    link: null
+    link:
```

Comment:
Not expected, but OK as the link is still suppressed (see https://endoflife.date/gstreamer).